distribution/changelog.txt

```diff
@@ -1680,4 +1681,4 @@ This is the first fully implemented version of OpenRCT2. RCT2.EXE is no longer r
 - Feature: RollerCoaster Tycoon 1 scenarios can now be opened in the scenario editor or by using the ‘edit’ command line action.
 - Feature: The ‘have fun’ objective can now be selected in the scenario editor.
 - Feature: Twitch integration.
-- Fix: Litter bins now get full and require emptying by handymen.
\ No newline at end of file
+- Fix: Litter bins now get full and require emptying by handymen.
```

Comment:
Please revert this accidental change. Your editor likely automatically added the EOL character at the end.

Reply:
That's strange...I'm not seeing anything on my end in the editor.

Reply:
Ok, I think it's gone now.